src/TodoList.js

```diff
@@ -0,0 +1,82 @@
+const addInIt = document.getElementById('addIng');
+const myTodoTitle = document.getElementById('input');
+const myDsisPlaydeList = document.getElementById('displayedlist');
+
+let ShowingmyData = [];
+const myPreserveData = (ShowingmyData) => {
+  localStorage.setItem('data', JSON.stringify(ShowingmyData));
+};
+const myDataGet = JSON.parse(localStorage.getItem('data'));
+if (myDataGet != null) {
+  ShowingmyData = myDataGet;
+}
+///  function
+const ShowiingMyData = () => {
+  let ShowiingMyData = '';
+  ShowingmyData.forEach((users, i) => {
+    ShowiingMyData += `<div class="myitem"><li class="mytolist"><input type="mycheckbox" class="mycheckbox" ${users.completed}>
+       <input type="text" id="sendOutMyData" class='sendOutMyData' value='${users.name}'></input></li><i class="material-icons" id="mydotico" onclick="myDataRemove(${i})">delete_all</i></div>`;
+  });
+  myDsisPlaydeList.innerHTML = ShowiingMyData;
+};
+
+ShowiingMyData();
+
+const myDataAdded = () => {
+  const myTodoIndex = ShowingmyData.length + 1;
+  ShowingmyData.push({ name: myTodoTitle.value, completed: false, indexx: myTodoIndex });
+  myPreserveData(ShowingmyData);
+  ShowiingMyData();
+  myTodoTitle.value = '';
+};
+
+addInIt.addEventListener('click', myDataAdded);
+// my enter press button function
+myTodoTitle.addEventListener('keytouch', (event) => {
+  if (event.key === 'Enter') {
+    myDataAdded();
+  }
+});
+/* eslint-disable 
+*/ // my remove function 
+myDataRemove = (id) => {
+  ShowingmyData.splice(id, 1);
+  myPreserveData(ShowingmyData);
+  ShowiingMyData();
+  for(let i = 0; i <ShowingmyData.length; i++) {
+    ShowingmyData[i].indexx = i + 1;
+    myPreserveData(ShowingmyData);
+   } 
+};
+const sendOutMyData = document.querySelectorAll('.sendOutMyData')
+const mycheckbox = document.querySelectorAll('.mycheckbox');
+for(let i = 0; i < ShowingmyData.length; i+=1) {
+
+  myBoxValue= () => {
+ if(mycheckbox[i].checked) {
+  console.log('checkbox is on')
+  ShowingmyData[i].completed = `check`;
+  sendOutMyData [i].style.textDecorationLine = 'border-line';
+  myPreserveData(ShowingmyData);
+
+
+ } else {
+  console.log('check off')
+  ShowingmyData[i].completed = `uncheck`;
+  sendOutMyData [i].style.textDecorationLine = 'none';
+  myPreserveData(ShowingmyData);
+
+ }
+};
+mycheckbox[i].addEventListener('click', myBoxValue)
+}
+
+
+
+for(let i = 0; i < ShowingmyData.length; i+=1) {
+  if(ShowingmyData[i].completed ===  `check`) {
+    sendOutMyData [i].style.textDecorationLine = 'border-line';
+  } else {
+    sendOutMyData [i].style.textDecorationLine = 'none';
```

Comment:
Kindly note that your project at this stage do not need all this functions and code. At this stage, you supposed to work on the following as shown in the project requirements: 

- [ ] Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:

     -   description [string].
     -   completed [bool].
     -   index: [number].

- [ ] Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- [ ] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- [ ] Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.